bin/hiera

```diff
@@ -243,6 +243,19 @@ unless options[:verbose]
   Hiera.logger = "noop"
 end
 
+# Allow lookup of keys with or without prepended double colons
```

Comment:
This doesn't seem necessary to fix the issue we've seen. I don't understand how the scope is used sufficiently to know whether there's a case we haven't thought of, but I lean towards doing the minimal fix until we come up with a case that needs this.